src/writing-tests-for-phpunit.rst

```diff
@@ -646,7 +646,7 @@ whether an exception is thrown by the code under test.
     {
         public function testException(): void
         {
-            $this->expectException(InvalidArgumentException::class);
+            $this->expectException(InvalidArgumentException::class);            
```

Comment:
I think the additional space should be removed.

Reply:
Fixed.